app.js

```diff
@@ -32,6 +31,8 @@ app.use((err, request, response, next) => {
     response.status(err.status).send({ message: err.message });
   } else if (err.code === "22P02") {
     response.status(400).send({ message: "Invalid Review ID" });
+  } else if (err.code === "23503") {
+    response.status(404).send({ message: "Review Not Found" });
   } else response.status(500).send({ message: "Internal Server Error" });
 });
```

Comment:
You can start to think about extracting these into their own file if you want!

Reply:
Yes will do!